### ISSUE TITLE
Yugoslavia - ML-161/LM-161 Minesweeper (Free Piston Gas Turbine)

### ISSUE THREAD

**Issue by nuk92** [PygmalionOfCyprus/cmo-db-requests#2338]
### DB Selector

Both

### Hypothetical

No

### Name

ML-161 Snažni (CWDB), ML-161 Gradac (DB3K), LM-161 Gradac (DB3k, refti 1&2)

### Class

Sirius Class

### Country

Yugoslavia

### Service

Navy

### In Commission

1961-1992

### Number in Service

1x

### Length (m)

46.65

### Beam (m)

8.73

### Draft (m)

2.6

### Standard Displacement (t)

325

### Full Displacement (t)

401

### Complement

36

### Max Sea State

5

### Primary Propulsion Type

Diesel

### Propulsion

2x Sigma-Pescara GS 34 Free Piston Engine, 2x Alsthom Belfort Gas Turbine, Free Piston Gas Turbine

### Fuel Capacity (kg)

37000

### Flank Speed (kt)

18

### Range (nm) at Speed (kt)

2548nm @ 14kts

### Sensors

1x DRBN-30 (CWDB)
------------------
1x Decca 1226 (DB3K)
1x Type 193M Sonar (DB3K, refit 1)
1x TSM 2022 Sonar (DB3K, refit 2)
1x Eyeball Mk1

### Mounts

1x Bofors 40mm/L60 (bow) (CWDB)
1x 20mm Single (stern) (CWDB)
----------------
2x 20mm M71 Single (bow & stern) (2100 rounds each) (DB3K)

### Aircraft & Docking Facilities

1x Docking Facility for 2x PAP-104 (refit 1 and 2)

### Comms/Datalinks

VHF (unsecure)
UHF (unsecure)
HF (unsecure)
VHF (secure)
UHF (secure)
HF (secure)

### Ship Codes

Onboard Degaussing Gear (Magnetometer), Wooden Hull Construction

### Comments

Basically the same class as the minesweepers in request: #2278
This one had a very unique Free Piston Gas Turbine propulsion and was much faster than it's sister ships.

### Sources

https://www.paluba.info/smf/index.php?topic=640.msg241219#msg241219
Ratni Brodovi Jugoslavenske Ratne Mornarice 1945-1991, Zvonimir Freivogel, Zagreb 2021, ISBN: 9789533660042, page 181
